.github/workflows/go.yml

```diff
@@ -0,0 +1,30 @@
+name: Go
+
+on:
+  push:
+    branches:
+      - '*'
+      - '**'
+      - '!master'
+  pull_request:
+    branches:
+      - '*'
+      - '**'
+      - '!master'
+
+jobs:
+  run_tests:
+    name: Run tests
+    runs-on: ubuntu-latest
+    steps:
```

Comment:
I think this part should be in another PR.

Reply:
This file triggers GitHub Actions on my fork repo.